analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/descriptions/Modifier.java

```diff
@@ -30,7 +30,7 @@ public int mask() {
   }
 
   public static Modifier valueOf(com.github.javaparser.ast.Modifier modifier) {
-    boolean onType = modifier.getParentNode().map(n -> n instanceof TypeDeclaration).orElse(false);
+    boolean onType = modifier.getParentNode().map(TypeDeclaration.class::isInstance).orElse(false);
```

Comment:
Is this really better/isn't this just the same thing?

Reply:
for me they both indeed do the same thing but sonarcloud likes the ::isInstance more. As long as no tests failed i decided to keep the change.